File: am4pa/runners/test/sample_generation/experiments/75_75_6_75_75/operand_generator.jl
using MatrixGenerator

function operand_generator()
    A::Array{Float64,2} = generate((75,75), [Shape.General, Properties.Random(-1, 1)])
    B::Array{Float64,2} = generate((75,6), [Shape.General, Properties.Random(-1, 1)])
    C::Array{Float64,2} = generate((6,75), [Shape.General, Properties.Random(-1, 1)])
    D::Array{Float64,2} = generate((75,75), [Shape.General, Properties.Random(-1, 1)])
    return (A, B, C, D,)
end

File: am4pa/runners/test/sample_generation/experiments/75_75_6_75_75/variants/Matlab/operand_generator.m
function [out] = operand_generator()
    import MatrixGenerator.*;
    out{ 1 } = generate([75,75], Shape.General(), Properties.Random([-1, 1]));
    out{ 2 } = generate([75,6], Shape.General(), Properties.Random([-1, 1]));
    out{ 3 } = generate([6,75], Shape.General(), Properties.Random([-1, 1]));
    out{ 4 } = generate([75,75], Shape.General(), Properties.Random([-1, 1]));
end